### PR TITLE
fix_normlize

### PR DESCRIPTION
**Pull request by fxyfxy777**
### fix_normlize

修正paddle.nn.functional.normalize算子后，还是有0.4%的mismatch，原因是paddle的精度更高，torch出现的inf和nan更多，且全部覆盖paddle出现inf/nan的位置

### CODE REVIEW

**Review by cangtianhuang — APPROVED** [review]
LGTM
